<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmvViewClass</class>
 <widget class="QWidget" name="EmvViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmvView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2" colspan="2">
    <widget class="QComboBox" name="configurationPicker">
     <property name="toolTip">
      <string>Select Configuration from list. Only available Configurations are shown.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="warningTag">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>warning</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="entityPicker">
     <property name="toolTip">
      <string>Select Entity from list. Add by pressing the Open File Button.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="entityLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Entity Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="configurationChangeButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Apply changes to selected configuration.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choose Configuration:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="openFilePushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Click to add new Entity to the list.</string>
     </property>
     <property name="text">
      <string>Open Entity File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Choose loaded Entity:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="languagePicker">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Choose display language of Entity, if supported. Auto will detect System Language.</string>
     </property>
     <item>
      <property name="text">
       <string>Auto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Deutsch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fran√ßais</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1" colspan="4">
    <widget class="QPushButton" name="openSettingsPushButton">
     <property name="toolTip">
      <string>Show EMV Settings</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
